exercises/practice/bottle-song/.formatter.exs

```diff
@@ -0,0 +1,4 @@
+# Used by "mix format"
+[
+  inputs: ["{mix,.formatter}.exs", "{.meta,config,lib,test}/**/*.{ex,exs}"]
```

Comment:
Let's keep this file standard, I understand why you added `.meta`, but that folder is not made available to students, so it could be a source of confusion.